server/src/main/java/org/candlepin/dto/service/BrandingDTO.java

```diff
@@ -0,0 +1,225 @@
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.dto.service;
+
+import org.candlepin.dto.TimestampedCandlepinDTO;
+import org.candlepin.service.model.BrandingInfo;
+
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+import java.util.Date;
+import java.util.Objects;
+
+/**
+ * A DTO representation of the Branding used in Service Adapter Layer.
+ */
+public class BrandingDTO extends TimestampedCandlepinDTO<BrandingDTO> implements BrandingInfo {
+
+    private String productId;
+    private String name;
+    private String type;
+
+    /**
+     * Initializes a new BrandingDTO instance with null values.
+     */
+    public BrandingDTO() {
+        // Intentionally left empty
+    }
+
+    /**
+     * Initializes a new BrandingDTO instance which is a shallow copy of the provided
+     * source entity.
+     *
+     * @param source
+     *  The source entity to copy
+     */
+    public BrandingDTO(BrandingDTO source) {
+        super(source);
+    }
+
+    /**
+     * Sets this branding's created date.
+     *
+     * @param created the created date to set on this branding DTO.
+     *
+     * @return this branding object
+     */
+    public BrandingDTO created(Date created) {
+        this.created = created;
+        return this;
+    }
+
+    /**
+     * Sets this branding's updated date.
+     *
+     * @param updated the updated date to set on this branding DTO.
+     *
+     * @return this branding object
+     */
+    public BrandingDTO updated(Date updated) {
+        this.updated = updated;
+        return this;
+    }
+
+    /**
+     * Sets this branding's product id.
+     *
+     * @param productId the product id to set on this branding DTO.
+     *
+     * @return this branding object
+     */
+    public BrandingDTO productId(String productId) {
+        this.productId = productId;
+        return this;
+    }
+
+    /**
+     * Returns this branding's product id.
+     *
+     * @return this branding's product id.
+     */
+    @Override
+    public String getProductId() {
+        return productId;
+    }
+
+    /**
+     * Sets this branding's product id.
+     *
+     * @param productId the product id to set on this branding DTO.
+     */
+    public void setProductId(String productId) {
+        this.productId = productId;
+    }
+
+    /**
+     * Sets this branding's name.
+     *
+     * @param name the name to set on this branding DTO.
+     *
+     * @return this branding object
+     */
+    public BrandingDTO name(String name) {
```

Comment:
Extraneous setter. Let's stick with the more standard set* and get*, but keep the chaining behavior.